addons/web_editor/static/src/js/wysiwyg/dialog.js

```diff
@@ -36,7 +37,7 @@ var WysiwygDialog = Dialog.extend({
 
         var self = this;
         this.opened(function () {
-            self.$('input:visible:first').focus();
+            self.$(options.focusField).focus();
```

Comment:
The option is called `focusField` but is used like it was something broader like `focusSelector` (which would be a weird option to have IMO.) I suggest the following:
```js 
const selector = options.focusField
    ? `input[name=${options.focusField}]` 
    : 'input:visible:first';
self.$(selector).focus();
```